### DISCUSSION TITLE
Is the protein level data log transformed?

### DISCUSSION THREAD

**Discussion by ArinehTahmasian** [vdemichev/DiaNN#1103] [General]
Hi there,
Thanks for your continuous support.
Just confirming if the values in report.pg_matrix.tsv are log transformed? 
thanks!

**Comment by vdemichev** [comment]
Hi, no pg_matrix is not log-transformed.

Best,
Vadim

**Reply by ArinehTahmasian** [reply]
Thanks for your reply.
